.gitignore

```diff
@@ -13,3 +13,5 @@ coverage.html
 .glide
 .scBuildImage
 .init
+.generate*
+**/zz*.go
```

Comment:
I think that is sufficient reason to check them in for now. I just woke up, and right now you get argument to authority. Please let's not try to be the tail that wags the dog on this yet. Going to be very hard to get help debugging if we don't do what upstream does.